tensorflow/cc/tools/freeze_saved_model.cc

```diff
@@ -201,15 +242,27 @@ Status FreezeGraphDef(const SavedModelBundle& saved_model_bundle,
     if (reachable_node_names.find(node.name()) == reachable_node_names.end()) {
       continue;
     }
+    StatusOr<string> handle_name;
```

Comment:
Hi Tamas, I got this comment from google reviewer. Would you please make this change?
This predeclaration gets confusing - better to do an extra "if" below to keep things within scope.

Reply:
I have added an extra if condition to scope the variable better.